.gitignore

```diff
@@ -13,3 +13,4 @@ cmake-build-*/
 # The Doxygen-generated documentation goes here:
 bigtable/doc/html/
 doc/html/
+gh-pages/
```

Comment:
Why is this a directory? This should be a branch name, so it shouldn't be in a `.gitignore`, unless I'm missing something.

Reply:
The script creates a temporary clone of the gh-pages branch in the gh-pages directory, and copies the (potentially new) documentation into it:

https://github.com/GoogleCloudPlatform/google-cloud-cpp/pull/154/files#diff-886613c4e549e681677337e465a9ed5dR40

Happy to name it something else if you think that is confusing.

Reply:
Ah, sorry, missed that entirely. Yes, please, let's call it something different like `docs-build` so that others are not confused reading this.